src/Data/String/CodePoints.purs

```diff
@@ -78,10 +78,11 @@ codePointFromInt n = Nothing
 -- | >>> codePointToInt (codePointFromChar 'B')
 -- | 66
 -- | 
--- | >>> boldA = codePointFromInt 0x1D400
+-- | >>> boldA = codePointFromInt 119808
 -- | >>> boldA
--- | Just (CodePoint 0x1D400)
```

Comment:
Rather than converting these two lines to use decimal, I think it would be better to leave these two lines as they were, and say `119808 -- is the same as 0x1D400` on the last line.